.github/workflows/ci.yaml

```diff
@@ -0,0 +1,24 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version:
+        - '2.6'
+        - '2.7'
+    env:
+      BUNDLE_PATH: scripts/vendor/bundle
```

Comment:
This whole scripts thing is just weird.  See also https://github.com/ManageIQ/manageiq-appliance-build/issues/158

Reply:
assuming that is for future PRs?

Reply:
yeah, this is too complicate to solve here.